public/apps/configuration/panels/audit-logging/code-editor.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ *   Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+import React, { useState } from 'react';
+import { EuiCodeEditor, EuiText, EuiTextColor } from '@elastic/eui';
+import { get } from 'lodash';
+
+import 'brace/theme/textmate';
+import 'brace/mode/json';
+
+export function CodeEditor(props: {
```

Comment:
Where is it used? I didn't see it in the screenshots

Reply:
This will be used for compliance settings. Let me remove this for now to avoid confusion and add back together with compliance settings.